docs/cadl-doc.md

```diff
@@ -65,7 +65,7 @@ Cadl:
 ```cadl
 alias Foo = int64;
 ```
-<span style="color: red; font-style:italic;"> Note: </span>
+<span style={{color: 'red',  fontStyle: 'italic'}}> Note: </span>
```

Comment:
Can you explain this change?

Reply:
Inline styling inside tables in markdown must conform to JSX for Docusaurus to process it. Still looking for other possibilities.